grades/models.py

```diff
@@ -173,6 +174,33 @@ def __str__(self):
         )
 
 
+class MicromastersProgramCertificate(TimestampedModel):
+    """
+    Model for storing MicroMasters program certificates
+    """
+    program = models.ForeignKey(Program, null=False)
+    user = models.ForeignKey(User, null=False)
+    hash = models.CharField(max_length=32, null=False, unique=True)
```

Comment:
Since this is generated from the program and user ids, can we make this a property instead of storing it in the database?

Reply:
Nevermind, let's leave this as is